src/vsl/parser/grammar/assignment.ne

```diff
@@ -2,16 +2,21 @@
 @include "primitives.ne"
 
 # Parses assignment statements
-AssignmentStatement -> AssignmentType _ TypedIdentifier ( _ "=" _ Expression {% nth(3) %}):? {%
-    (d, l) => new t.AssignmentStatement(
-        d[0] === "let" ? t.AssignmentType.Constant : t.AssignmentType.Variable,
-        d[2],
-        d[3],
-        l
-    )
-%}
+AssignmentStatement
+   -> AssignmentType _ TypedIdentifier ( _ "=" _ Expression {% nth(3) %}):? {%
+        (data, location) => {
+            var type;
+            if (data[0] === "let") {
```

Comment:
I'll make this lookup-object thing a const variable above `AssignmentStatement` for now. If you want to you can change it later before or after merging. I don't have write access so you can merge it when you have time.